common/inbus/inbus.py

```diff
@@ -88,14 +88,14 @@ def subject_versions(
 
 
 def schedule_subject_by_version_id(
-    subject_version_id: dto.SubjectVersionId,
+    subject_version_id: dto.SubjectVersionId, inbus_subject_id: int
```

Comment:
```suggestion
    subject_version_id: dto.SubjectVersionId, inbus_semester_id: int
```

Reply:
Thanks for spotting! It needs to be changed also a few lines below as well.